data/virt_autotest/host_unattended_installation_files/autoyast/dev_host_12.xml.ep

```diff
@@ -42,10 +42,10 @@
       <secure_boot>false</secure_boot>
       <terminal>console</terminal>
       % if ($check_var->('SYSTEM_ROLE', 'xen')) {
-      <xen_append>loglevel=5 vt.color=0x07 splash=silent console=hvc0 <%= defined $bmwqemu::vars{"OPT_KERNEL_PARAMS"} ? $bmwqemu::vars{"OPT_KERNEL_PARAMS"} : "" %></xen_append>
-      <xen_kernel_append><%= defined $bmwqemu::vars{"XEN_SERIAL_CONSOLE"} ? $bmwqemu::vars{"XEN_SERIAL_CONSOLE"} : "com2=115200,8n1 console=com2" %>,vga loglvl=all guest_loglvl=all sync_console</xen_kernel_append>
+      <xen_append>vt.color=0x07 splash=silent console=hvc0 <%= defined $bmwqemu::vars{"ENABLE_CONSOLE_KERNEL_LOG"} ? "loglvl=all guest_loglvl=all" : "loglvl=debug guest_loglvl=debug" %> <%= defined $bmwqemu::vars{"OPT_KERNEL_PARAMS"} ? $bmwqemu::vars{"OPT_KERNEL_PARAMS"} : "" %></xen_append>
```

Comment:
If it's allowed to customize the level of kernel log , it would be fine to use var `ENABLE_CONSOLE_KERNEL_LOG`. Otherwise, since it is fixed level of `all`, maybe making it reflected in to var name will be better.

Reply:
There is no need to specify all in variable name. Enabling it just means all is wanted, otherwise there is no need to enable it. Keep current name is enough and clear.

Reply:
Well, its initial impression to me is different... I do not what exact level of log it is enabling. Neither the difference with the original log collection. I get them after checking details. Maybe other users will have similar feeling.
Not a big deal, up to you to decide. Just raise it.

Reply:
There is no need. ```ENABLE_CONSOLE_KERNEL_LOG``` is fairly enough. Lengthy variable name does not help too much for this case.